projects/data-structures-algorithms-sprint.md

```diff
@@ -28,10 +28,8 @@ This sprint will also help you practice study skills, summarizing, teamwork, and
 ### Schedule
 
 #### Day 1
-In the morning, the Program Facilitator will give a quick introduction to data structures, split you into groups, and assign two data structures and three algorithms topics per group.Thegroup members compare notes about topics, start creating introductory slides, and decide who will present what. groups will plan out demos, select beginner exercises and resources, polish slides, and practice presenting their introductory topic overviews (remember that reading is discouraged and eye contact and a loud, confident voice are encouraged).
-
-Homework: read about your assigned data structures topics, highlight or outline the most important parts, and review runtime complexity ([topic outline](https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md), [slides](https://drive.google.com/open?id=1ZcOdekB_aP59huZdp4X0u6EfUJKgxzK7y8LqCmzSLC8), [video](https://drive.google.com/open?id=1ZoHxJMUiKOKPqu69vX3b_aeYGlDlRL6n)). 
-Groups will prepare introductory overviews to topics in this order:
+In the morning, the Program Facilitator will give a quick introduction to data structures, split you into groups, and assign two data structures and three algorithms topics per group. Homework: read about your assigned data structures topics, highlight or outline the most important parts, and review runtime complexity ([topic outline](https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md), [slides](https://drive.google.com/open?id=1ZcOdekB_aP59huZdp4X0u6EfUJKgxzK7y8LqCmzSLC8), [video](https://drive.google.com/open?id=1ZoHxJMUiKOKPqu69vX3b_aeYGlDlRL6n)). 
```

Comment:
What’s happening in this file?

Reply:
an unwanted change was noticed which was not done by me. So from the previous commit, I have changed that.

Reply:
I think your branch may be old.  Please get the current version of the file from this repo and add your changes to that one. It's still making changes in this PR that are not yours.

Reply:
@alodahl  I have already updated my branch with that of the master.

Reply:
So the changes for lines 31-34 are yours?

Reply:
Can you check the master version of this file and see if we need the honework section? It’s being removed here.